volatility3/framework/symbols/linux/__init__.py

```diff
@@ -483,6 +483,22 @@ def get_module_from_volobj_type(
 
         return kernel
 
+    @classmethod
+    def convert_fourcc_code(cls, code: int) -> str:
```

Comment:
I think adding a new method to `LinuxUtilities` is a MINOR bump please (and then updating the requirements of the plugin that needs this function).

Reply:
I've bumped the required numbers, it should be good right now.